jvb/src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -1169,7 +1173,9 @@ public IQ healthCheckIqReceived(@NotNull HealthCheckIQ iq)
         /**
          * The total number of times the layering of an incoming video stream changed (updated on endpoint expiration).
          */
-        public AtomicInteger totalLayeringChangesReceived = new AtomicInteger();
+        public CounterMetric layeringChangesReceived = VideobridgeMetricsContainer.getInstance().registerCounter(
+                "layering_changes_received",
+                "Number of times the layering of an incoming video stream changed (updated on endpoint expiration).");
```

Comment:
FYI: I would like to move to a model where we increment these live insted of on expire(), but that's probably outside the scope of your project